server/handler.py

```diff
@@ -50,10 +71,15 @@ def query_cache_for_date(dt: datetime.datetime):
 
 
 def insert_data_to_cache(tle_data: str, dt: datetime.datetime):
+    """Insert data into cache."""
     S3.put_object(Body=tle_data.encode(), Key=_dt_hash(dt))
 
 
-def hello(event, context, auth_client=SpaceTrackClient, query_call=query_cache_for_date):
+def hello(
```

Comment:
This function name is embarrassingly left over from a serverless template :man_facepalming: